qa/rpc-tests/wallet.py

```diff
@@ -105,13 +107,14 @@ def run_test (self):
         self.nodes[1].sendrawtransaction(txns_to_send[0]["hex"], True)
         self.nodes[1].sendrawtransaction(txns_to_send[1]["hex"], True)
 
-        # Have node1 mine a block to confirm transactions:
-        self.nodes[1].generate(1)
+        # Have node1 mine a 7 block to confirm transactions:
+        self.nodes[1].generate(7)
+        time.sleep(2)
```

Comment:
This sleep too.

Reply:
This test very unstable. Sometimes need to wait till transactions will be received by node. Need to investigate how to improve this test. I think now is a bad time to uncomment it in common regtests file.